packages/strapi-helper-plugin/lib/src/components/OverlayBlocker/index.js

```diff
@@ -73,7 +73,7 @@ class OverlayBlocker extends React.Component {
       <div className="buttonContainer">
         <a
           className={cn('primary', 'btn')}
-          href="https://strapi.io/documentation/configurations/configurations.html#server"
+          href="https://strapi.io/documentation/v3.x/getting-started/introduction.html"
```

Comment:
If we are just going to direct them to the documentation (and not the specific page) then we should just set the link to `https://strapi.io/documentation` which will prevent further issues if the path changes.

Reply:
@derrickmehaffy  Damn. My mistake. Yah that would definitely prevent future issues. Updating the requested change.